requirements.txt

```diff
@@ -0,0 +1,4 @@
+.ignore==1.7.6
```

Comment:
@qaivanpleten please remove this file from this branch. This is a separate task and it has to be implemented in the separate branch.